src/helpers/interfaces.ts

```diff
@@ -1,6 +1,8 @@
 import type {AxiosPromise, AxiosRequestConfig} from 'axios'
 import type {Writable} from 'stream'
 
+import {TOOL_GENERATOR_NAME, TOOL_GENERATOR_VERSION} from 'src/commands/sbom/constants'
```

Comment:
This kind of import doesn't work in TS, I wonder why it even worked tbh 😁 And I didn't see it during review.

First, the import isn't relative here. And `src/` only works in the repo. Once packaged in NPM, we only have the `dist/` folder containing JS files compiled from TS.

Also, since we are defining global types in `src/helpers/interfaces` from these constants, they should be global too. Otherwise it's a circular dependency (command -> global -> command).

I'm creating a PR because it's breaking this build in another repo: https://github.com/DataDog/synthetics-ci-github-action/pull/301